webpack.config.js

```diff
@@ -248,6 +248,14 @@ module.exports = {
           },
         ],
       },
+      {
+        test: /\.png$/i,
+        use: [
+          {
+            loader: 'url-loader',
```

Comment:
I think it'd be better to do use the `file-loader` for this rather than the `url-loader` to decrease the total size.

Reply:
I attempted to use `file-loader` but could not get it to emit a working URL (it was emitting `img/filename.png` (with base dir being `/css`) instead of `/img/filename.png`) so I went with this instead. I think it's fine to inline that image, it's just 60kB or so.

Reply:
Actually, I think I've figured it out.

Reply:
Done. I've created `public/img/webpack` so we have a directory for images output by webpack that can be cleanly deleted every time.